Documentation/config.txt

```diff
@@ -1467,6 +1467,11 @@ commit.verbose::
 	A boolean or int to specify the level of verbose with `git commit`.
```

Comment:
The Git project takes pride in its helpful commit messages. To that end, we have a couple of guidelines:

- wrap the commit message at <=76 columns per line
- start the commit message by a single line (like a header)
- start the first line with a prefix to set the mood, in your case probably `commit:`
- use the imperative form, in your case `add a setting to allow empty messages by default`
- then, describe in one or more paragraphs the background, e.g. from where you got this idea, what the motivation is to have empty messages, what the downsides are, thoughts on the implementation if you have any, etc

See e.g. 9bf5d4c4e22fce46f87e576b2ae09982e926bfc5 if you want to follow a good example